python-clusters/create-gke-cluster/cluster.json

```diff
@@ -25,18 +25,49 @@
             "type": "STRING",
             "defaultValue": "latest"
         },
+        {
+            "name": "s-network",
+            "type": "SEPARATOR",
+            "label": "Networking",
+            "description": "<ul><li>For VPC-native clusters, leave IP ranges empty to let GKE automatically assign them.</li></ul>"
```

Comment:
the comment appears far from the 2 relevant fields, so either move it to their description, or put it in the longDescription of the plugin, or rephrase